fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewGrade.kt

```diff
@@ -0,0 +1,196 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.shopping.ui
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.CornerSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import mozilla.components.ui.colors.PhotonColors
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+private val height = 24.dp
+private val borderColor = Color(0x26000000)
+private val reviewGradeAColor = Color(0xFF10AD56)
+private val reviewGradeBColor = Color(0xFF007DEC)
+private val reviewGradeCColor = Color(0xFFF4A902)
+private val reviewGradeDColor = Color(0xFFF27313)
+private val reviewGradeFColor = Color(0xFFD51235)
+private val reviewGradeAColorExpanded = Color(0xFFDBF3E6)
+private val reviewGradeBColorExpanded = Color(0xFFD9ECFC)
+private val reviewGradeCColorExpanded = Color(0xFFFDF2D9)
+private val reviewGradeDColorExpanded = Color(0xFFFDEADC)
+private val reviewGradeFColorExpanded = Color(0xFFF9DBE1)
```

Comment:
do we need this as vals in the file now that they're inside the enum class?

Reply:
I think we should keep them as they are in reality "constants" and allows the enums to reference them with semantic names, so when they change, we just update these vals, let me know what you think!

Reply:
I'm thinking about this more from a file bloat perspective, but I totally understand your point and think it's valid and could help prevent accidental color regressions down the road. Maybe it'll be less noticeable when ReviewGrade is moved out of this Composable file. 

:shipit: